charts/datahub/subcharts/datahub-frontend/values.yaml

```diff
@@ -4,6 +4,26 @@
 
 replicaCount: 1
 
+hpa:
+  enabled: false # If enabled, Horizontal Pod Autoscaler (HPA) will automatically scale the number of replicas based on CPU and memory utilization thresholds.
+  minReplicas: 1
```

Comment:
Usually recommend the default number of replicas in an HA (high availability) deployment to be 2.

This is important as it ensures the frontend deployment can always service requests even during rollouts such as programmed restarts, configuration changes & upgrades.

It's not the default value in `replicaCount` simply because the community usually use the defaults for testing, not for production ready set ups.  In this case it wouldn't make sense to have an HPA system scale the deployment to 1 if the replicaCount is set to 2 (recommended in production deployments).